src/borg/archiver.py

```diff
@@ -938,20 +939,33 @@ def format_cmdline(cmdline):
             if not archive_names:
                 return self.exit_code
 
+        archives = []
         for i, archive_name in enumerate(archive_names, 1):
             archive = Archive(repository, key, manifest, archive_name, cache=cache,
                               consider_part_files=args.consider_part_files)
             stats = archive.calc_stats(cache)
-            print('Archive name: %s' % archive.name)
-            print('Archive fingerprint: %s' % archive.fpr)
-            print('Comment: %s' % archive.metadata.get('comment', ''))
-            print('Hostname: %s' % archive.metadata.hostname)
-            print('Username: %s' % archive.metadata.username)
-            print('Time (start): %s' % format_time(to_localtime(archive.ts)))
-            print('Time (end):   %s' % format_time(to_localtime(archive.ts_end)))
-            print('Duration: %s' % archive.duration_from_meta)
-            print('Number of files: %d' % stats.nfiles)
-            print('Command line: %s' % format_cmdline(archive.metadata.cmdline))
+
+            archiveSummary = archive.get_summary()
+            statsSummary = stats.get_summary()
+            cacheSummary = cache.get_summary()
```

Comment:
Nit pick: [PEP8](https://www.python.org/dev/peps/pep-0008/) names would be preferred (eg. `archive_summary` instead of `archiveSummary`)